.github/workflows/candig-testing.yml

```diff
@@ -90,3 +83,38 @@ jobs:
                 tmp/vault_audit.log
                 tmp/ingest
                 tmp/index
+
+      - name: Clone CanDIG Playwright Runbook
```

Comment:
I think it would be good to add all the Playwright tests as a separate job rather than all under Build-CanDIG